packages/ui/src/theme/utils.ts

```diff
@@ -34,20 +34,22 @@ function referenceValue(value: string) {
   return value;
 }
 
-export function cssValue(token: DesignToken<{ value: unknown }>) {
+export function cssValue(token: DesignToken<any>) {
```

Comment:
Don't we have some idea of the shape of the DesignToken?

Reply:
Updated it to be `BaseDesignToken` because that is what we are calling `cssValue` with in `createTheme.ts`